topics/ecology/tutorials/openrefine_gbif/tutorial.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: tutorial_hands_on
+
+title: Cleaning GBIF data using OpenRefine
+zenodo_link: ''
+questions:
+- How can I use OpenRefine to clean data?
+- How do I check and clean biodiversity data using OpenRefine?
+
+objectives:
+- Use OpenRefine faceting functionnalities to apply mass editing and manage duplicates
+- Use OpenRefine clustering and filtering functionnalities to edit, transform data notably using regular expression
+- Use OpenRefine to apply API services on your data
+time_estimation: 0h45m
+key_points:
+- OpenRefine is a powerfull tool, with many functionnalities to check, clean and enrich your data
+contributors:
```

Comment:
@yvanlebras  this should fix the linting error

```suggestion
contributions:
```

Reply:
Oh! THANK YOU so much Saskia!